### PR TITLE
UI Fix & Time Workaround

### PR DESCRIPTION
**Pull request by qzhsjz**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -8,7 +8,7 @@
     "clean": "ionic-app-scripts clean",
     "build": "ionic-app-scripts build --prod",
     "ionic:build": "ionic-app-scripts build",
-    "ionic:serve": "ionic-app-scripts serve --no-open --port 8102",
+    "ionic:serve": "ionic-app-scripts serve --no-open --port 8100",
```
Comment (awmleer) [inline package.json]: 这个还是改回8102吧。。
```suggestion
    "ionic:serve": "ionic-app-scripts serve --no-open --port 8102",
```

**Comment by qzhsjz** [comment]
font{
        line-height: 1.6;
    }
    ul,ol{
        padding-left: 20px;
        list-style-position: inside;
    }


    
    
        行
        
    
    
        
            
        
    
    
     
    
        
                    
            
                
                    
                            
                                
                            
                            
                                qzhsjz
                            
                    
                        
                            
                                    qzhsjz@gmail.com
                            
                        
                
            
        
        
    
    
        签名由
        网易邮箱大师
        定制
    
 

    On 12/3/2018 22:32，awmleer<notifications@github.com> wrote： 


@awmleer commented on this pull request.







In package.json:

> @@ -8,7 +8,7 @@

     "clean": "ionic-app-scripts clean",

     "build": "ionic-app-scripts build --prod",

     "ionic:build": "ionic-app-scripts build",

-    "ionic:serve": "ionic-app-scripts serve --no-open --port 8102",

+    "ionic:serve": "ionic-app-scripts serve --no-open --port 8100",



这个还是改回8102吧。。

⬇️ Suggested change

-    "ionic:serve": "ionic-app-scripts serve --no-open --port 8100",

+    "ionic:serve": "ionic-app-scripts serve --no-open --port 8102",







—You are receiving this because you were assigned.Reply to this email directly, view it on GitHub, or mute the thread.

File: src/services/labreserve.service.ts [hunk 2]
```diff
@@ -33,8 +33,8 @@ export class LabreserveService {
     console.log(endTime);
     console.log(description);
     let data = {
-      startTime: new Date(startTime).getTime(),//this.datePipe.transform(startTime, 'yyyy-MM-dd hh:mm'),
-      endTime: new Date(endTime).getTime(),//this.datePipe.transform(endTime, 'yyyy-MM-dd hh:mm'),
+      startTime: new Date(startTime).getTime() + (3600000 * -8),//this.datePipe.transform(startTime, 'yyyy-MM-dd hh:mm'),
```
Comment (awmleer) [inline src/services/labreserve.service.ts]: 这两处也先改回去吧。。时区的问题我去处理

Comment (awmleer) [reply src/services/labreserve.service.ts]:
```suggestion
      startTime: new Date(startTime).getTime(),//this.datePipe.transform(startTime, 'yyyy-MM-dd hh:mm'),
```

File: src/services/labreserve.service.ts [hunk 3]
```diff
@@ -33,8 +33,8 @@ export class LabreserveService {
...
+      endTime: new Date(endTime).getTime() + (3600000 * -8),//this.datePipe.transform(endTime, 'yyyy-MM-dd hh:mm'),
```
Comment (awmleer) [inline src/services/labreserve.service.ts]:
```suggestion
      endTime: new Date(endTime).getTime(),//this.datePipe.transform(endTime, 'yyyy-MM-dd hh:mm'),
```

**Review by awmleer — APPROVED** [review]

**Pull request merged**